File: src/main/java/com/movisens/xs/api/models/MonitoringAlert.java
package com.movisens.xs.api.models;

import com.google.gson.annotations.Expose;

public class MonitoringAlert extends Monitoring {
    @Expose
    private boolean isWarning = false;

    public MonitoringAlert(long probandId, String date, String name, String message, boolean includeInMail, boolean isWarning) {
        super(probandId, date, MonitoringType.ALERT, name, message, includeInMail);
        this.isWarning = isWarning;
    }
}

File: src/main/java/com/movisens/xs/api/models/MonitoringAlert.kt
package com.movisens.xs.api.models

import com.google.gson.annotations.Expose
import com.movisens.xs.api.models.Monitoring.MonitoringType

class MonitoringAlert(
    probandId: Long,
    date: String,
    name: String,
    message: String,
    includeInMail: Boolean,
    @field:Expose val isWarning: Boolean = false
) : Monitoring(probandId, date, MonitoringType.ALERT, name, message, includeInMail)